src/model/AirLoopHVACOutdoorAirSystem.cpp

```diff
@@ -69,6 +71,20 @@ namespace model {
 
   namespace detail {
 
+    // Helpers
```

Comment:
My first thought was that it was odd to see this seemingly general purpose helper function down here in the oa system implementation, but now that I think about what is going on I guess the context matters in terms of what "outletNode" is, so I guess it makes sense to have this function here.